### PR TITLE
Greatly improve the Finnish translation

### PR DESCRIPTION
**Pull request by mary-kate**
Title Case is not used in Finnish, using it looks just off.

The previous translation had many typos and inconsistent terminology overall, e.g. CPU is (as per [the Finnish Wikipedia](https://fi.wikipedia.org/wiki/Suoritin), for example), "suoritin", or if you insist on using the Anglicism, "prosessori" but never "processori".

"build" in the computing sense (a particular release of a computer software (e.g. the Windows operating system) program written in a compiled language) is "koontikäännös", which is slightly more precise than plain "versio".

"RAM" (Random Access Memory) is often left untranslated but there's a good translation for it, too: [keskusmuisti](https://fi.wikipedia.org/wiki/Keskusmuisti)

Some messages had untranslated English.

### CODE REVIEW

**Comment by micwoj92** [comment]
Please update version to 2.3.1.0 and add yourself to translators. @NabsiYa Do you agree with the changes?

**Review by NabsiYa — APPROVED** [review]
well yeah i do agree i kinda rushed the orginal translation so i didnt add great detail to it.

File: Langs/040B.lang [hunk 1]
```diff
@@ -4,56 +4,56 @@ Translator=NabsiYa
...
-64 Bit CPU=64-Bit:n Processori
-64 Bit OS=64-Bit:n Käyttöjärjestelmä
-32 Bit CPU=32-Bit:n Processori
-32 Bit OS=32-Bit:n Käyttöjärjestelmä
-Unable to Determine=Ei Voida Määrittää
-Unable to Check List=Ei Voida Katsoa Listaa
-Error Accessing List=Ongelma Avatessa Listaa
-Not Currently Listed as Compatible=Ei Tällä Hetkellä Tuoettu
-Listed as Compatible=Listattu Tuetuksi
+64 Bit CPU=64-bittinen suoritin
```
Comment (NabsiYa) [inline Langs/040B.lang:24]: also i wanna mention specially that i was told this was "incorrect" so i used "Bit:n" instead which annoys me upto this day so thanks for correcting it.

**Review by NabsiYa — APPROVED** [review]

File: Langs/040B.lang (hunk 1 above)
Comment (micwoj92) [reply Langs/040B.lang:24]: I didn't mean that "bittinen" was incorrect, I meant that you used x64 and x32, sorry for not being clearer.

**Comment by PriitUring** [comment]
Shouldn't "Translation by" (current Line 11) be translated instead of writing username?

**Comment by micwoj92** [comment]
Oh, I didn't even notice that, thanks for notifying!

**Pull request merged**